docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -94,8 +94,22 @@ Every PR is building website and on success it shows the link to preview.
 
 We use [Netlify](https://www.netlify.com/) for hosting. We are using Open Source license (PRO). Thanks Netlify for this!
 
-On every commit to `master` netlify runs CI that invokes `make web` (defined in [netlify.toml](https://github.com/improbable-eng/thanos/blob/master/netlify.toml)))
```

Comment:
We need this prefix IMO (similar to any resource outside of `docs`)

Reply:
Where does this not work for you? I could be missing something obvious, sorry for that! :D 

On the website itself, this works fine -> https://deploy-preview-1176--thanos-io.netlify.com/contributing/how-to-contribute-to-docs.md/ 
and in markdown https://github.com/improbable-eng/thanos/blob/6a5b9b41856fa8781f3bb352322422cc25538aa2/docs/contributing/how-to-contribute-to-docs.md#deployment

Reply:
ignore me you just fixed the processing script as well (: Thanks ;p That's way more consistent :+1: